Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y gcc make
+
+COPY . /app
```

Comment:
This COPY isn't needed anymore because we're mounting the directory

Reply:
Yup, should be good now!